asreview/entry_points/gui.py

```diff
@@ -1,26 +1,42 @@
+import logging
+
 from asreview.entry_points.base import BaseEntryPoint
 from asreview.entry_points.base import _base_parser
 
 
-DESCRIPTION_ORACLE = """
+DESCRIPTION_LAB = """
 ASReview LAB - Active learning for Systematic Reviews.
 """
 
 HOST_NAME = "localhost"
 PORT_NUMBER = 5000
 
 
-class GUIEntryPoint(BaseEntryPoint):
+class LABEntryPoint(BaseEntryPoint):
     description = "Graphical user interface for ASReview."
 
+    def _pre_execute(self):
```

Comment:
This can now be removed, right?

Reply:
Yes. Thanks. I was starving..

Reply:
Yes, me too!

Reply:
Resolved. Successfully merged branch 'pizza' into master.